### ISSUE TITLE
SkeGD code

### ISSUE THREAD

**Issue by lilianzhi0** [JunfLi-TJU/OKL-Hinge#1]
Can you provide the code for SkeGD please?

**Comment by JunfLi-TJU** [comment]
Yes, I have uploaded the code of SkeGD contained in OKS-SkeGD-20190716.zip. If you have any questions about the experiments, please feel free to ask me.
